### ISSUE TITLE
Replace R² metric with NRMSE in model evaluation

### ISSUE THREAD

**Issue by WuuuuuuX** [WuuuuuuX/beach#1]
## Changes Required

1. Remove R² (R-squared) metric calculations
2. Add NRMSE (Normalized Root Mean Square Error) calculations
3. Update the output logging

## Code Changes

Replace the current metrics calculation with:

```python
rmse_list = []
nrmse_list = []

for test_file in glob.glob(os.path.join(test_folder, "*.xlsx")):
    dataset = ExcelDataset(test_file)
    X_test = torch.cat([dataset.features, dataset.x], dim=1).to(device)
    y_test = dataset.y.squeeze(1).to(device)
    x_plot = dataset.x.squeeze(1).cpu().numpy()

    model.eval()
    likelihood.eval()
    with torch.no_grad(), gpytorch.settings.fast_pred_var():
        pred = likelihood(model(X_test))
        mean = pred.mean.cpu().numpy()
        lower, upper = pred.confidence_region()
        lower = lower.cpu().numpy()
        upper = upper.cpu().numpy()
        y_true = y_test.cpu().numpy()

    # Optional: Sort by x
    sort_idx = x_plot.argsort()
    x_plot = x_plot[sort_idx]
    y_true = y_true[sort_idx]
    mean = mean[sort_idx]
    lower = lower[sort_idx]
    upper = upper[sort_idx]

    # Calculate RMSE and NRMSE
    rmse = np.sqrt(mean_squared_error(y_true, mean))
    # NRMSE = RMSE / (y_max - y_min)
    nrmse = rmse / (np.max(y_true) - np.min(y_true))
    
    rmse_list.append(rmse)
    nrmse_list.append(nrmse)
    print(f"[{os.path.basename(test_file)}] RMSE: {rmse:.4f} | NRMSE: {nrmse:.4f}")

# Output average RMSE and NRMSE
mean_rmse = np.mean(rmse_list)
mean_nrmse = np.mean(nrmse_list)
print(f"Average RMSE across {len(rmse_list)} tests: {mean_rmse:.4f}")
print(f"Average NRMSE across {len(nrmse_list)} tests: {mean_nrmse:.4f}")
```

## Implementation Notes

1. NRMSE is calculated by normalizing RMSE by the range of the observed data
2. This gives a relative error metric that is more comparable across different scales
3. Remove all r2_score imports and calculations
